src/assets/i18n/en.json5

```diff
@@ -2124,6 +2126,36 @@
 
   "home.top-level-communities.help": "Select a community to browse its collections.",
 
+  "info.accessibility-settings.breadcrumbs": "Accessibility settings",
+
+  "info.accessibility-settings.disableNotificationTimeOut.label": "Hide notifications automatically",
+
+  "info.accessibility-settings.disableNotificationTimeOut.hint": "When this toggle is activated, notifications will remain until manually closed.",
+
+  "info.accessibility-settings.liveRegionTimeOut.label": "Live region time-out",
+
+  "info.accessibility-settings.liveRegionTimeOut.hint": "The duration after which a message in the live region disappears.",
+
+  "info.accessibility-settings.liveRegionTimeOut.unit": "Seconds",
```

Comment:
I'd remove these "Seconds" keys, as they are not very accessible.  This should be part of the label itself, so that screen readers understand the input field is representing seconds.